support-frontend/assets/pages/[countryGroupId]/components/checkoutSummary.tsx

```diff
@@ -174,10 +197,7 @@ export default function CheckoutSummary({
 					}
 					headerButton={
 						showBackButton && (
-							<BackButton
-								path={`/${geoId}${productDescription.landingPagePath}`}
```

Comment:
It's interesting that we didn't try to preserve any abTest param before? Do we definitely need to do that now?